backend/internal/proxy/datasource.go

```diff
@@ -36,7 +38,14 @@ func ProxyDatasource(c *gin.Context) {
 
 	targetURL := c.Param("path")
 
-	client := &http.Client{}
+	client := &http.Client{
+		Transport: &http.Transport{
+			DialContext: (&net.Dialer{
+				Timeout:   time.Duration(time.Minute * 3),
```

Comment:
这里的建立连接超时时间和 keepalive 的时间是不是写反了。。。

Reply:
// Timeout is the maximum amount of time a dial will wait for
// a connect to complete. If Deadline is also set, it may fail
// earlier.
//
// The default is no timeout.
//
// When using TCP and dialing a host name with multiple IP
// addresses, the timeout may be divided between them.
//
// With or without a timeout, the operating system may impose
// its own earlier timeout. For instance, TCP timeouts are
// often around 3 minutes.
这里是TCP连接超时没有写反。。哈哈哈哈我还去翻了一下go注释

Reply:
好的，明白，不过从业务角度(代理请求到相应的数据源服务器，例如 prometheus , jaeger)来看，3分钟建立连接超时时间长了些，30s 到 1分钟比较合理 :D

同理，KeepAlive 在 1 - 2分钟也比较合理，因为这个产品主要面向内网环境，可以适当长一些。

Reply:
我是不是应该让它加载配置文件 给默认值

Reply:
倒也不必，这里配置的意义不是太大，以后有需要再改 :D

Reply:
我已改完

Reply:
额。。。你可能要解决下冲突了，我修改了目录结构

Reply:
我看是的 我重新提交了一个